platform-ui-skin/src/main/webapp/skin/less/core/helpers.less

```
@@ -474,6 +474,14 @@
   width: 100% !important;
   min-width: 100% !important;
 }
+.full-window-height {
+  height: calc(var(--100vh), 100vh) !important;
+  min-height: calc(var(--100vh), 100vh) !important;
```

The usage of `~"..."` is to ensure that `calc` isn't interpreted as LESS command but CSS.

```suggestion
  height: ~"var(--100vh, 100vh) !important";
  min-height: ~"var(--100vh, 100vh) !important";
```

---

done